docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
+- Travis Build runs without errors (green)
+- Meaningful javadoc comments for:
+  - All classes
+  - All non-private methods and fields
+  - Private methods with non-obvious implementation
```

Comment:
> Private methods with non-obvious implementation

This line was not intended to request implementation details in javadoc comments, but rather to allow omitting of javadoc comments for methods that do not require them such as get() and set() methods, just like @hnicke described. It makes sense to just allow omitting javadoc for get/set etc. for all methods and not just for private methods like it was initially described. So it would probably be better to just request javadoc for all methods, regardless of visibility, like suggested by @mklopp.